docs/evaluation/how_to_guides/create_few_shot_evaluators.mdx

```diff
@@ -2,49 +2,22 @@
 
-:::tip Recommended Reading
-Before learning how to create few-shot evaluators, it might be helpful to learn how to setup automations (both online and offline) and how to leave corrections on evaluator scores:
-
-- [Set up online evaluations](../../../observability/how_to_guides/online_evaluations)
-- [Bind an evaluator to a dataset in the UI (offline evaluation)](./bind_evaluator_to_dataset)
-- [Audit evaluator scores](./audit_evaluator_scores)
-
-:::
+This guide covers how to apply corrections to feedback scores to enable few-shot examples. 
 
 ## Create your evaluator
 
-:::tip
-The default maximum few-shot examples to use in the prompt is 5. Examples are pulled randomly from your dataset (if you have more than the maximum).
-
-:::
-
-When creating an [online](../../../observability/how_to_guides/online_evaluations) or [offline](./bind_evaluator_to_dataset) evaluator - from a tracing project or a dataset, respectively - you will see the option to use corrections as few-shot examples. Note that these types of evaluators
-are only supported when using mustache prompts - you will not be able to click this option if your prompt uses f-string formatting. When you select this,
-we will auto-create a few-shot prompt for you. Each individual few-shot example will be formatted according to this prompt, and inserted into your main prompt in place of the `{{Few-shot examples}}`
-template variable which will be auto-added above. Your few-shot prompt should contain the same variables as your main prompt, plus a `few_shot_explanation` and a score variable which should have the same name
-as your output key. For example, if your main prompt has variables `question` and `response`, and your evaluator outputs a `correctness` score, then your few-shot prompt should have `question`, `response`,
-`few_shot_explanation`, and `correctness`.
-
-You may also specify the number of few-shot examples to use. The default is 5. If your examples will tend to be very long, you may want to set this number lower to save tokens - whereas if your examples tend
-to be short, you can set a higher number in order to give your evaluator more examples to learn from. If you have more examples in your dataset than this number, we will randomly choose them for you.
-
-![Use corrections as few-shot examples](./static/use_corrections_as_few_shot.png)
-
-Note that few-shot examples are not currently supported in evaluators that use Hub prompts.
-
-Once you create your evaluator, we will automatically create a dataset for you, which will be auto-populated with few-shot examples once you start making corrections.
+Before learning how to apply corrections, [set up your LLM-as-a-judge evaluator to use corrections](/evaluation/how_to_guides/llm_as_judge). 
```

Comment:
feels like the explanation for this should be in this guide. adding corrections is the easy part of "how to set up few shot evaluators". the key is the setting up